Mbrookhuis ssh push init 2 (#622)
uyuni-project/uyuni-docs: modules/client-configuration/pages/contact-methods-pushssh.adoc

----
+
OPTIONAL: You can remove the [command]``--tunnel`` option, if you do not want to use tunneling.

> +
> OPTIONAL: When [command]``ssh_push_sudo_user`` has been defined, add the option [command]``--notty``. This will allow the administrator to enter the ``root`` password. 

Reviewer: ```suggestion
. OPTIONAL: When you have defined [command]``ssh_push_sudo_user``, you can allow use of the root password by adding the [command]``--notty`` option.
```

Reviewer: This makes it its own step, rather than tacking it on the previous one, and rewords it to be more active.
